utilities/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const CONTENTFUL_API_ACCESS_TOKEN = 'Bearer qVGC4H5ZmObAQMKyniPJNtNjtGkfmhi_OgVVy9jmtR8'
+export const CONTENTFUL_API_URL = 'https://graphql.contentful.com/content/v1/spaces/zqvr29h6rh51/'
+export enum HttpRequestMethod {
```

Comment:
Can we insert a blank line here to make clear that one group is `const` and one is `enum`?